packages/vue-renderer/src/renderers/spa.js

```diff
@@ -149,7 +152,9 @@ export default class SPARenderer extends BaseRenderer {
     // Set meta tags inside cache
     this.cache.set(cacheKey, content)
 
-    return content
+    // Return a copy of the content, so that future
+    // modifications do not effect the data in cache
+    return cloneDeep(content)
```

Comment:
content is not cached (and meta is already cloned) so do we need to clone it again?

Reply:
content is cached on line 153, when we return this to be used later, any changes to content is reflected inside the cache.